src/Core/Content/Flow/Dispatching/Action/ChangeCustomerGroupAction.php

```diff
@@ -12,7 +12,7 @@
 /**
  * @internal
  */
-#[Package('services-settings')]
+#[Package('discovery')]
 class ChangeCustomerGroupAction extends FlowAction implements DelayableAction
```

Comment:
I'm not exactly sure how far we should go. This is part of the flowbuilder, but of course touched our topic.

But on the other hand, we are also claming the seo-elements in categories, which is the same case but we're on the other side. Is it good that we consequently just take everything?

Reply:
That’s an excellent question!

I believe the answer depends on the context:

Regarding the Flow Builder:
The Flow Builder is a system provided by Shopware, and it’s possible that we could extend it with domain-specific flows, such as a CmsElementRenderedFlow in the future.

Regarding SEO elements in categories:
The primary focus here is our domain. Maintaining a single SEO element is more manageable for us since we possess the overarching knowledge. On the other hand, it would likely be more challenging for the SEO team to fully understand categories and maintain their SEO element effectively.